location/utils.py

```diff
@@ -54,3 +54,48 @@ def approx_home_location(data, sr_col='stay_region'):
     home = Counter(trg_data[sr_col]).most_common()[0][0]
 
     return home
+
+
+def compute_total_gyration(data, sr_col='stay_region'):
+    """
+    Computes the total radius of gyration.
+
+    Parameters:
+    -----------
+
+    data: DataFrame
+        Location data.
+
+    sr_col: str
+        Column name for stay region.
+        Default is 'stay_region'.
+
+    Returns:
+    --------
+
+    (radius of gyration): float
+        Radius gyration in meter.
+    """
+    loc_data = data.copy()
+    loc_data = loc_data[pd.notnull(loc_data[sr_col])]
+    # different visited locations
```

Comment:
Could you confirm that we should only consider unique points? If someone goes to a particular place very often, the radius of gyration would be very different, right?